test/test.vcxproj

```diff
@@ -32,6 +32,14 @@
   <ImportGroup Label="Shared" />
   <ImportGroup Label="PropertySheets" />
   <PropertyGroup Label="UserMacros" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">
+    <OutDir>$(SolutionDir)$(Platform)\$(Configuration)\</OutDir>
+    <IntDir>$(Platform)\$(Configuration)\</IntDir>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|Win32'">
+    <OutDir>$(SolutionDir)$(Platform)\$(Configuration)\</OutDir>
+    <IntDir>$(Platform)\$(Configuration)\</IntDir>
+  </PropertyGroup>
```

Comment:
Any idea why this was added? These look like they might be default values.

I think the NAS2D project file may have inconsistent settings here between configurations. May be worth checking the NAS2D project file for configuration problems here, and if it relates to any configuration problems in the test project.

Reply:
For running the test executable, the path to the executable will be impacted by this setting.

In particular, the line:
`%APPVEYOR_BUILD_FOLDER%\%PLATFORM%\%CONFIGURATION%\test.exe`

The above settings are trying to place the executable in a subfolder from the solution directory. I assume that's the folder with the solution file, and not the repository root. The repository root will be `%APPVEYOR_BUILD_FOLDER%\`.

Reply:
Before clearing these settings, compare OP2Utility's test project settings. By default x86 builds will use directories `Debug` and `Release`. x64 builds will use `x64\Debug` and `x64\Release` directories. 

We had to standardize by telling the project to use a hardcoded `x86` directory when building. Otherwise it was difficult to line up the location of the test executable between the x86 and x64 builds. 

I was trying to use $(Platform). For x86 builds it produces `win32` for the directory. Unfortunately, appveyor produces `x86`, hence our hardcode of x86 in OP2Utility. 

From Outpost2UtilityTest project settings:
<img width="600" alt="OP2UtilityTestProjectSettings" src="https://user-images.githubusercontent.com/15183337/56166323-53b21b00-5fa3-11e9-98e4-dca5637a3ea0.png">

For the output directory, instead of using $(SolutionDir) we could use $(ProjectDir), which might look a bit cleaner.

Reply:
Ok, yes, I remember that now. Thank you for the reminder.

In terms of the "Win32", I think that's because the `Platform` setting in the project file is literally "Win32". We can probably change that to "x86", as it's just an identifying text string.

I suspect newly created projects will use "x86", but because the project was upgraded from an older version of Visual Studio, it's still using the old default of "Win32".

Reply:
@DanRStevens 
@Brett208

I found this thread on the AppVeyor support pages while looking into testing a personal project:
https://help.appveyor.com/discussions/problems/5282-tests-not-being-run#comment_40766291 

Apparently the `Win32` vs `x86` definition of the `Platform` variable is related to native vs managed projects. To fix it you can re-define the platform variable to the correct `Win32` value in the `.appveyor.yml` file:

> `before_test:`
> `  - cmd: if %Platform%=x86 set Platform=Win32`

and still keep the actual project property page as `$(Platform)`

Reply:
@cugone, Looks like a nice alternative.